testutils/data.go

```diff
@@ -37,8 +37,8 @@ var (
 
 func checkCacheSize(n int) {
 	// Start with a reasonably large cache.
-	if n < 8 {
-		n = 8
+	if n < 1024 {
```

Comment:
Since we cache a bunch of random bytes and reuse them, we were likely to return the same bytes when using something like `RandBytes(10)` where we have a small cache we select from. This caused thrift-gen tests to sometimes share the same directory name, making it harder to debug failures.